packages/core/src/lib/icon/icon.svelte

```diff
@@ -68,6 +68,7 @@ $: {
   viewBox="0 0 24 24"
   aria-hidden="true"
   focusable="false"
+  {name}
```

Comment:
This significantly improves our testability by allowing for `container.querySelector('svg[name=camera-outline]');`

I have not found any obvious accessibility problems with doing this but if there are any, I am happy to revert this and find some other way of testing the svgs.

Reply:
Wow! Good callout--I should've checked MDN... will do so in the future. Thank you for your feedback and suggestion.

I really would like to add explicit icon tests because Kurt and I ran a few icon bugs while developing our searchable select (that we fixed, but are now happy to have tests for). (we ran into an issue where the icon was still rendering but wasn't changing and we wouldn't be able to test that. Not a big deal. )

I swapped to your data-icon-name fix in c698011

If you feel strongly about this or this would cause accessibility issues, I am happy go back to 
```ts
// 2 icons in the dropdown + one in the search + chevron in search
expect(querySelectorAll('svg')).toHaveLength(4)
```
But I really like:
```ts
expect(
   container.querySelector('svg[data-icon-name=viam-process]')
).toBeInTheDocument();
expect(
  container.querySelector('svg[data-icon-name=language-cpp]')
).toBeInTheDocument();
```